pom.xml

```diff
@@ -344,6 +343,24 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>java-11</id>
+            <activation>
+                <jdk>[11,17)</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>8</maven.compiler.release>
+            </properties>
+        </profile>
+        <profile>
+            <id>java-17</id>
+            <activation>
+                <jdk>[17,</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>8</maven.compiler.release>
+            </properties>
+        </profile>
```

Comment:
I do not get why is this needed if at the end the settings are anyway the same. Can't you just keep `<maven.compiler.release>8</maven.compiler.release>` in the main properties?

Reply:
Ok, thanks,